.circleci/ci/src/workflows/workflow-pull-requests.ts

```diff
@@ -223,7 +223,7 @@ export class PullRequestsWorkflow {
           name: 'Lint & test APIM Console',
           context: config.jobContext,
           'apim-ui-project': config.dockerImages.console.project,
-          resource_class: 'large',
+          resource_class: 'xlarge',
```

Comment:
From my analysis we had a Job with between 15 and 25 minutes before. With the changes of this commit and the xlarge we are more on an execution time of ~8 minutes. 
This divides the time by 2 and should not change the use of credit (as an xlarge is 2 times more expensive than a large) 💭

Reply:
I think there will be some flaky ones left, but that's already better.